### PR TITLE
chore: update to the latest version and update docs

### PR DESCRIPTION
**Pull request by verbanicm**

### CODE REVIEW

File: .github/workflows/handle.yml [hunk 1]
```diff
@@ -29,11 +29,11 @@ jobs:
       contents: 'read'
       id-token: 'write'
       pull-requests: 'write'
-    uses: 'abcxyz/access-on-demand/.github/workflows/handle.yml@v0.1.5' # Change here to pin the tag or commit.
```
Comment (sqin2019) [inline .github/workflows/handle.yml:32]: Should we add labeled trigger event?

File: .github/workflows/note.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ jobs:
   note:
     permissions:
       pull-requests: 'write'
-    uses: 'abcxyz/access-on-demand/.github/workflows/note.yml@v0.1.5' # Change here to pin the tag or commit.
```
Comment (sqin2019) [inline .github/workflows/note.yml:34]: Should we add labeled trigger event?

File: .github/workflows/handle.yml (hunk 1 above)
Comment (verbanicm) [reply .github/workflows/handle.yml:32]: ah good catch!

File: .github/workflows/note.yml (hunk 2 above)
Comment (verbanicm) [reply .github/workflows/note.yml:34]: ah good catch!

**Review by sqin2019 — APPROVED** [review]

**Pull request merged**